app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java

```diff
@@ -734,6 +735,13 @@ public int getPointerColor() {
         return SettingsStore.getInstance(this).getPointerColor();
     }
 
```

Comment:
can we add a method for [`getDeviceType()`](https://github.com/MozillaReality/FirefoxReality/pull/1097/files#diff-94535c76764a3a4f9a3d7cbf496c0d7dR21) than returns a string for each (e.g., `"unknown"`, `"oculus-go"`, etc.)? or should I just import `DeviceType` directly?

Reply:
You should just be using `DeviceType.getDeviceType()` directly.